elftools/elf/elffile.py

```diff
@@ -135,11 +135,13 @@ def get_section_index(self, section_name):
             self._make_section_name_map()
         return self._section_name_map.get(section_name, None)
 
-    def iter_sections(self):
+    def iter_sections(self, type=None):
```

Comment:
Please expand the method comment to explain what the `type` param means

Reply:
Yep, I missed that, thanks!